### ISSUE TITLE
/var/lib/lxcfs paths are not set correctly in lxc.mount.hook during ./configure

### ISSUE THREAD

**Issue by brauner** [lxc/lxcfs#11]
On the `lxcfs` portion of the `lxc` homepage it states that one should run:    

    git clone git://github.com/lxc/lxcfs
    cd lxcfs
    ./bootstrap.sh
    ./configure
    make
    sudo mkdir -p /var/lib/lxcfs
    sudo ./lxcfs -s -f -o allow_other /var/lib/lxcfs/

However, when running `./configure` the paths in `lxc.mount.hooks` are either prefixed with the default `/usr/local` (e.g. in `lxc.mount.hooks`: `/usr/local/var/lib/lxcfs/proc`) or when specifying the prefix in the `./configure` stage as in:

    ./configure --prefix=/usr

They get set to `/usr/var/lib/proc` which does not even exist. Maybe it would be good to  mention that `./configure` needs to be run as:

    ./configure --prefix=/usr --localstatedir=/var

**Comment by stgraber** [comment]
So you're really either meant to "make install" in which case the paths will be valid or manually use it in which case you should be able to set the right paths.

I don't like including instructions that will mess with the user's system like that.

**Comment by stgraber** [comment]
I'll be updating the website to mention that we do have packages nowadays and that people should use those unless they really know what they're doing.

**Comment by brauner** [comment]
I did both to check:

(1) `sudo make install` were all the files get put in `/usr/local`. This invalidates the instructions on your website (just confusing for users who might not have done this before). They should then read:

    sudo mkdir -p /usr/local/var/lib/lxcfs
    sudo ./lxcfs -s -f -o allow_other /usr/local/var/lib/lxcfs/

and in the container config you would need:

    lxc.include = /usr/local/share/lxc/config/common.conf.d/00-lxcfs.conf

(2) And I also did

    git clone git://github.com/lxc/lxcfs
    cd lxcfs
    ./bootstrap.sh
    ./configure --prefix=/usr --localstatedir=/var --sbin=/usr/bin
    make

In any case I think it would be good to mention that paths might need to be adjusted, and that packages are the preferred option.

Of course I get the point about not wanting to mess with the users system.

Thanks!
